gio/src/file_attribute_info_list.rs

```diff
@@ -0,0 +1,98 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::FileAttributeInfoList;
+
+use glib::translate::*;
+
+use std::fmt;
+use std::mem;
+use std::ptr;
+
+glib::wrapper! {
+    #[doc(alias = "GFileAttributeInfo")]
+    pub struct FileAttributeInfo(BoxedInline<ffi::GFileAttributeInfo>);
```

Comment:
should be on a `file_attribute_info.rs` file for rustdoc-stripper to pick it up. Otherwise it won't be documented

Reply:
Didn't know that, thanks :) Changed accordingly.

Reply:
I tend to run embed-docs and then strip them to see if a manual code is going to be documented. Pretty handy